bin/i18n/sync-in.rb

```diff
@@ -176,6 +230,7 @@ def get_i18n_strings(level)
       teacher_markdown
       placeholder
       title
+      start_html
```

Comment:
I don't think we want start_html added like this anymore

Reply:
I tried merging with staging to solve the conflicts and It has messed up the PR.